docs/includes/apiargs-MongoDBClient-method-startSession-option.yaml

```diff
@@ -0,0 +1,43 @@
+arg_name: option
+name: causalConsistency
+type: boolean
+description: |
+  Enables or disables :ref:`causal consistency <causal-consistency>` for the
+  session. :method:`Mongo.startSession()` enables ``causalConsistency`` by
+  default. After starting a session, you cannot modify its
+  ``causalConsistency`` setting.
+interface: phpmethod
+operation: ~
+optional: true
+---
+source:
+  file: apiargs-common-option.yaml
+  ref: readConcern
+replacement:
+  resource: "database"
+  parent: "client"
```

Comment:
[`MongoDB\Driver\Manager::startSession()`](http://php.net/manual/en/mongodb-driver-manager.startsession.php) only supports one option. The shell API differs in that its session is basically a context object (like client, database, collection). For drivers, the session is just a value object that we pass around.